src/scripts/itemdetailpage.js

```diff
@@ -252,7 +252,7 @@ define(["loading", "appRouter", "layoutManager", "connectionManager", "cardBuild
         var imgUrl, screenWidth = screen.availWidth,
             hasbackdrop = !1,
             itemBackdropElement = page.querySelector("#itemBackdrop"),
-            usePrimaryImage = "Video" === item.MediaType && "Movie" !== item.Type && "Trailer" !== item.Type || item.MediaType && "Video" !== item.MediaType;
+            usePrimaryImage = ("Video" === item.MediaType && "Movie" !== item.Type && "Trailer" !== item.Type) || (item.MediaType && "Video" !== item.MediaType) || ("MusicAlbum" === item.Type) || ("MusicArtist" === item.Type);
```

Comment:
Care to explain why second part checks for `item.MediaType` without any comparison? Is it just to check if it is present at all (like not `undefined`)?

Reply:
I think yes: not every item object has the MediaType attribute.
Probably removing the check of `item.MediaType` would also work as a fix of this particular issue.

The right thing to do should be fixing the API endpoints to return consistent objects...